lib/test/index.html

```diff
@@ -1,21 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <meta charset="utf-8">
-  <title>Ramda Tests</title>
-  <link rel="stylesheet" href="mocha.css" />
-</head>
-<body>
-  <h1>Tests for ramda</h1>
-  <div id="mocha"></div>
-  <script src="lib/test/expect.js"></script>
-  <script src="mocha.js"></script>
-  <script src="assert-shim.js"></script>
-  <!--[if lt IE 9]>
-    <script src="../lib/sauce/ie8-shim.js"></script>
-  <![endif]-->
-  <script>mocha.setup('bdd')</script>
-  <script src="bundle.js"></script>
-  <script src="../sauce/mocharun.js"></script>
-</body>
-</html>
```

Comment:
Understood, and I don't use it that way much any more.  It's just a little nostalgic tug.  But there is something that I do actually regret.  For some time a user could clone or download Ramda HEAD and simply use `ramda.js` or `ramda.min.js`.  She could also see the tests in action by opening that browser test file.  She didn't have to have any tools beyond git (or even unzip) to be able to use Ramda.  Granted she could not run a build herself.  But it was easily available to unsophisticated developers.

We broke some of this when we stopped including these files in the commits.  While I agreed to that we should do so, I still hope one day to get Travis to add this; I've dropped the ball on this, though.